app/main.py

```diff
@@ -1,3 +1,82 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | str | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance") -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: Distance | int | float) -> Distance:
```

Reply:
fix annotation everywhere

Reply:
Completely forgot about those, thank you. 😅